services/keywords/extract.py

```diff
@@ -0,0 +1,53 @@
+from google.cloud import language  # pylint: disable=import-error
+from google.cloud.language import enums, types  # pylint: disable=import-error
+import constants
+
+
+def gcloud_syntax_extraction(text):
+    """
+    Calls the gcloud natural langauge api's analyze syntax
+    Attributes:
+        text: the text which we wish to analyze
+    Returns:
+        the response given by the api call
+    """
+    try:
+        client = language.LanguageServiceClient()
+        # pylint: disable=no-member
+        document = types.Document(
+            content=text,
+            type=enums.Document.Type.PLAIN_TEXT)
+        response = client.analyze_syntax(document=document)
+    except Exception:
```

Comment:
You can remove the try/catch block anywhere you just re-raise the exception. Exceptions will actually automatically get re-raised all the way up the call stack until either the first function to try/catch it or the program's entrypoint (main). 

Normally, you'd only want to try/catch/re-raise if you wanted to add more context to the error message or if you tried to handle the error and failed, which you do not need in this case.